.storybook/preview.js

```diff
@@ -1,5 +1,15 @@
 import { INITIAL_VIEWPORTS } from '@storybook/addon-viewport';
 import { Breakpoints } from '@/client/models/Style';
+import React from 'react';
+import { FocusStyleManager } from '@guardian/source-foundations';
+
+export const FocusManagerDecorator = (storyFn) => {
+  React.useEffect(() => {
+    FocusStyleManager.onlyShowFocusOnTabs();
+  });
```

Comment:
as this doesn't have a dependency array, this will apply on every re-render, do we want that to happen, or do we just want it to run once/when the dependencies change?

Reply:
```js
  React.useEffect(() => {
    FocusStyleManager.onlyShowFocusOnTabs();
  }, []);
```

surely this would stop the re-render issue then?